cluster/src/main/java/io/scalecube/cluster/membership/MembershipProtocolImpl.java

```diff
@@ -569,6 +572,7 @@ private Message prepareSyncDataMsg(String qualifier, String cid) {
                       metadataStore.updateMetadata(member, metadata);
                       sink.next(MembershipEvent.createAdded(member, metadata));
                     })
+                .onErrorResume(TimeoutException.class, e -> Mono.empty())
```

Comment:
`TimeoutException` is expected one and must not pollute log with _Received unexpected error_   logs.